GameServerLib/Logic/Game.cs

```diff
@@ -58,8 +58,8 @@ Logger logger
         public void Initialize(Address address, string blowfishKey)
         {
             _logger.LogCoreInfo("Loading Config.");
-
-            Config = new Config(Program.ConfigPath);
+            
```

Comment:
This still needs addressing (you have whitespace on that empty line, if you're wondering what's up)